extension/encoding/awscloudwatchmetricstreamsencodingextension/extension_test.go

```diff
@@ -26,7 +26,7 @@ func TestNew_OpenTelemetry10(t *testing.T) {
 	require.NotNil(t, e)
 
 	_, err = e.UnmarshalMetrics([]byte{})
-	assert.EqualError(t, err, `failed to unmarshal metrics as 'opentelemetry1.0' format: 0 metrics were extracted from the record`)
+	assert.NoError(t, err)
```

Comment:
ditto

Reply:
Perfect suggestion, there was a bug lying around that wasn't caught yet.